gf-server-api/src/model/Room.ts

```diff
@@ -45,25 +45,17 @@
  *             description: The room's promo image URL.
  *             example: https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg
  *           slider:
- *             type: string
- *             description: The room's slider 1 image URL.
- *             example: https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg
- *           slider2:
- *             type: string
- *             description: The room's slider 2 image URL.
- *             example: https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg
- *           slider3:
- *             type: string
- *             description: The room's slider 3 image URL.
- *             example: https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg
- *           slider4:
- *             type: string
- *             description: The room's slider 4 image URL.
- *             example: https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg
- *           slider5:
- *             type: string
- *             description: The room's slider 5 image URL.
- *             example: https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg
+ *             type: array
+ *             items:
+ *              type:string
+ *             example: [
+ *               https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg,
+ *               https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg,
+ *               https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg,
+ *               https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg,
+ *               https://uploads-ssl.webflow.com/5e38379f9141a20d42ea7ab3/618177a1165e33f6ca935321_506.jpg,
+ *             ]
+ *             description: The rooms slider images URL.
```

Comment:
We should not save swagger specifications as comments near business models.

Reply:
I just change slider to array of slides